src/express/commands/restart.js

```diff
@@ -12,15 +12,25 @@ export async function restartAll(n) {
 
     let doc = await yaml.load(fs.readFileSync('./configs/devnet/remote-setup-config.yaml', 'utf8'));
     let borUsers = splitToArray(doc['devnetBorUsers'].toString())
+    let nodeIps = []
+    let hostToIndex = new Map()
```

Comment:
just for consistency, can we name this `hostToIndexMap` ?

Reply:
Fixed: [b658a55](https://github.com/maticnetwork/matic-cli/pull/86/commits/b658a55711f29b38b146399cdecd63cb4ecab326)